.ci/create_pkg.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+BRTG=$1
+BUILD_ID=$2
+IMG_VER=$3
+
+SRDIR="/etc/xlio_pkg/"
+RDIR="/etc/kubelet.d/"
+
+cd $SRDIR
+
+rcont=`crictl ps -s running -q --name xlio`
```

Comment:
Is it a requirement to use background run in the container? Maybe, more convenient to use a foreground run like "crictl exec .." on the running container?

Reply:
Will be reviewed in next step as improvement